projects/plugins/boost/app/modules/critical-css/class-critical-css.php

```diff
@@ -981,10 +981,14 @@ public function describe_provider_key( $provider_key ) {
 	 * Dismiss Critical CSS recommendations.
 	 */
 	public function dismiss_recommendations() {
-		check_ajax_referer( self::AJAX_NONCE, 'nonce' );
-		$response = array(
-			'status' => 'ok',
-		);
+		if ( check_ajax_referer( self::DISMISS_CSS_RECOMMENDATIONS_NONCE, 'nonce' ) && current_user_can( 'manage_options' ) ) {
+			$response = array(
+				'status' => 'ok',
+			);
+		} else {
+			$error = new \WP_Error( 'authorization', __( 'You do not have permission to change this status.', 'jetpack-boost' ) );
```

Comment:
I think the error message can be slightly improved given that they are not changing any status(I know I was made the same mistake). How about, "You do not have permission to take this action."?